internal/configs/version2/nginx-plus.virtualserver.tmpl

```diff
@@ -113,8 +113,11 @@ server {
         {{ $snippet }}
         {{ end }}
 
-        {{ with $l.Redirect }}
-        return {{ .Code }} "{{ .URL }}";
+        {{ with $l.Return }}
+            {{ if $l.DefaultType }}
+        default_type {{ $l.DefaultType }};
```

Comment:
{{ $l.DefaultType }} should be enclosed in `"..."` . otherwise, any space characters in the field in the YAML will cause problems